MaterialComponentsExamples.podspec

```diff
@@ -1,6 +1,7 @@
 experimental_sources = [
   'components/*/examples/experimental/supplemental/*.{h,m,swift}',
   'components/*/examples/experimental/*.{h,m,swift}',
+  'components/private/State/src/*.{h,m,swift}'
```

Comment:
Should all the state stuff go here or should it go in `components/schemes/` with all the other subsystem stuff?

Reply:
Yes, I moved it to components/schemes in the alpha version (see #6585) which is going to land in develop.